packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -45,6 +48,14 @@ export class GuxFormFieldColor {
   @Prop()
   labelPosition: GuxFormFieldLabelPosition;
 
+  /**
+   * Field indicator mark which can show *, (optional) or blank
+   * Defaults to required, this shows * for required fields and blank for optional
+   * When set to optional, shows (optional) for optional and blank for required.
```

Comment:
Maybe a slight change to be:

`When set to optional, the component will display (optional) for optional and blank for required.`